hazelcast/src/main/java/com/hazelcast/spi/impl/servicemanager/impl/ServiceManagerImpl.java

```diff
@@ -127,6 +128,7 @@ private void registerCoreServices() {
         registerService(SplitBrainProtectionServiceImpl.SERVICE_NAME, nodeEngine.getSplitBrainProtectionService());
         registerService(WanReplicationService.SERVICE_NAME, nodeEngine.getWanReplicationService());
         registerService(EventServiceImpl.SERVICE_NAME, nodeEngine.getEventService());
+        registerService(ConfigurationService.SERVICE_NAME, nodeEngine.getConfigurationService());
```

Comment:
Configuration service is already registered [here](https://github.com/hazelcast/hazelcast/blob/master/hazelcast/src/main/java/com/hazelcast/spi/impl/NodeEngineImpl.java#L177). This would be duplicate under different name. I would recommend using `ClusterWideConfigurationService.SERVICE_NAME` everywhere.